exporter/loadbalancingexporter/README.md

```diff
@@ -24,7 +24,10 @@ Refer to [config.yaml](./testdata/config.yaml) for detailed examples on using th
 * The `resolver` accepts either a `static` node, or a `dns`. If both are specified, `dns` takes precedence.
 * The `hostname` property inside a `dns` node specifies the hostname to query in order to obtain the list of IP addresses.
 * The `dns` node also accepts an optional property `port` to specify the port to be used for exporting the traces to the IP addresses resolved from `hostname`. If `port` is not specified, the default port 4317 is used.
-
+* The `routing_key` property is used to route spans to exporters based on different parameters. This functionality is currently enabled only for `trace` pipeline types. It supports one of the following values:
+    * `service`: routes spans based on its service name. This will be useful when you don't want to duplicate `service.name` in all exporter hosts. 
```

Comment:
Looking at this alone, I don't quite understand the message.

Reply:
Right now, we have trace-collector pods which will receive traces and have two exporters one for tail sampling and the other one for span metrics.

For tail sampler, we don't really face any issues but for span metrics because it's traceID routing, we see duplicate metrics for each service name on every host and it's crashing our pods with OOMKilled status

Reply:
Can you change the readme text to make it clearer? What you wrote above makes it clear to me; perhaps a more succinct version would do. Like:

```suggestion
    * `service`: routes spans based on their service name. This is useful when using processors like the span metrics, so all spans for each service are sent to consistent collector instances for metric collection. Otherwise, metrics for the same services are sent to different collectors, making aggregations inaccurate. 
```